tests/common/helpers/platform_api/fan.py

```diff
@@ -0,0 +1,74 @@
+""" This module provides interface to interact with the fan of the DUT
+    via platform API remotely """
+
+import json
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def fan_api(conn, index, name, args=None):
+    if args is None:
+        args = []
+    conn.request('POST', '/platform/chassis/fan/{}/{}'.format(index,
+                                                              name), json.dumps({'args': args}))
```

Comment:
This doesn't do much for line length. Suggest going back to single line, even if it's a bit long, it's more readable.

Reply:
fixed